tmc-langs-util/src/main/java/fi/helsinki/cs/tmc/langs/util/Main.java

```diff
@@ -0,0 +1,161 @@
+package fi.helsinki.cs.tmc.langs.util;
+
+import com.google.common.base.Optional;
+import fi.helsinki.cs.tmc.langs.ExerciseDesc;
+import fi.helsinki.cs.tmc.langs.RunResult;
+import fi.helsinki.cs.tmc.stylerunner.validation.ValidationResult;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+
+    final static String EXERCISE_PATH = "exercisePath";
+    final static String OUTPUT_PATH = "outputPath";
+    private static TaskExecutor executor = new TaskExecutorImpl();
+
+    public Main(TaskExecutor executor) {
+        Main.executor = executor;
+    }
```

Comment:
The rest of the methods here are static. Why is the method for setting the executor a constructor, not a static setter?

Alternatively, make everything non-static and have main() do only something like `new Main(new TaskExecutorImpl()).run(args)`.